spec/iex/resources/dividends_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe IEX::Resources::Dividends do
+  context 'known symbol' do
+    context 'with defaults', vcr: { cassette_name: 'dividends/msft' } do
+      subject do
+        IEX::Resources::Dividends.get('MSFT', '6m')
+      end
+      let(:dividends) { subject.first }
+      it 'retrieves dividends' do
+        expect(dividends.payment_date).to eq '2018-03-08'
+        expect(dividends.record_date).to eq '2018-02-15'
+        expect(dividends.declared_date).to eq '2017-11-29'
+        # expect(dividends.amount).to eq 0.42
```

Comment:
@dblock Please, uncomment this to test the behavior I mentioned on the previous comment.